data/json/itemgroups/guns.json

```diff
@@ -207,6 +208,7 @@
       [ "ak47", 100 ],
       [ "ak74", 60 ],
       [ "an94", 40 ],
+      [ "bh_m89", 60],
```

Comment:
In that case I'd rather leave the big game rifle weights alone in both cases and leave re-balancing to a separate PR.